conda.bzl

```diff
@@ -26,6 +26,28 @@ CONDA_INSTALLER_FLAGS = {
     "Linux": ["-b", "-f", "-p", "{}"],
 }
 
+MINIFORGE_MAJOR = "3"
+MINIFORGE_MINOR = "4.10.3-7"
+MINIFORGE_SHA = {
+    "Windows": {
+        "x86_64": "e3b1e7c5a02315c90bbb20d27614e00183ba8247594c57fb1f0484ccf5f9471c",
+    },
+    "MacOSX": {
+        "x86_64": "a25c1b381b20873ed856ce675a7a2ccf48f1d6782a5cdce9f06496e6ffa7883f",
+        "arm64": "3cd1f11743f936ba522709eb7a173930c299ac681671a909b664222329a56290",
+    },
+    "Linux": {
+        "x86_64": "4de9b7dcc9b2761136f4a7a42a8b2ea06ae2ebc61d865c9fca0db3d6c90b569d",
+        "aarch64": "d597961defe8c7889f3e924d0dc7624fab2c8845abccdd8ffa8da8018ff3dc6e",
+        "ppc64le": "8825827240c0d06413876055bf3a04d8704f0e5ac773692a352502862dce7aa5",
+    },
+}
+
+# TODO(jiawen): It's trivial to replace "Miniforge" with "Mambaforge".
```

Comment:
As using `mamba` is orthogonal to using `miniforge` let's just forget about `mambaforge`. If the user wants to use `miniforge` and `mamba` he would specify that he wants to use the `miniforge` installer and that he wants to use `mamba` (after #22). That way we are keeping the decision about installer and about whether to use `conda` or `mamba` separate and independent.

Reply:
Reverted comment.